packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.8+14
+
+* Check captureSession before call to stopRepeating in pausePreview
```

Comment:
This needs to follow https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
It should also describe the effect of the change, not the mechanics of the change. In this case, since the goal is to fix a crash, this should explain that it fixes a crash (and explain in what circumstances, from a plugin client perspective, that crash could happen).

Reply:
Changed descriprion

Reply:
Test added